keps/sig-security/1753-logs-sanitization/kep.yaml

```diff
@@ -15,4 +17,4 @@ approvers:
   - "@dashpole"
 prr-approvers:
   - "@wojtek-t"
```

Comment:
re-assign to me please, wojtek is over-subscribed

Also, when this is ready for PRR, please add the approval file as described here: https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md#submitting-a-kep-for-production-readiness-approval

Enhancements freeze is SOON: 2/9. If you want this to make 1.21 please get it ready for review asap.

Reply:
@johnbelamaric this PR won't require any PRR review, just moving directories :) We are not targeting this for 1.21.

Thanks for being so on top of this!

Reply:
ok!